app-modules/ai/database/migrations/2025_05_19_195232_seed_permissions_add_qna_advisor_permission.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2025, Canyon GBS LLC. All rights reserved.
+
+    Advising App™ is licensed under the Elastic License 2.0. For more details,
+    see https://github.com/canyongbs/advisingapp/blob/main/LICENSE.
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+use CanyonGBS\Common\Database\Migrations\Concerns\CanModifyPermissions;
+use Illuminate\Database\Migrations\Migration;
+
+return new class () extends Migration {
+    use CanModifyPermissions;
+
+    /** @var array<string, string> */
+    private array $permissions = [
+        'qna_advisor.view-any' => 'QnA Advisors',
+        'qna_advisor.create' => 'QnA Advisors',
+        'qna_advisor.*.view' => 'QnA Advisors',
+        'qna_advisor.*.update' => 'QnA Advisors',
+        'qna_advisor.*.delete' => 'QnA Advisors',
+        'qna_advisor.*.restore' => 'QnA Advisors',
+        'qna_advisor.*.force-delete' => 'QnA Advisors',
```

Comment:
After re-reading the ticket. Let's make sure the permission group is named "QnA Advisor", singular

Reply:
Do we know if the QnA feature will have multiple configurable advisors or not? We might need this information to decide if the permission needs to be like this or not

Reply:
I believe it will.